dynamic-app/src/main/java/com/okta/idx/android/dynamic/auth/DynamicAuthViewModel.kt

```diff
@@ -190,15 +212,20 @@ internal class DynamicAuthViewModel(private val recoveryToken: String) : ViewMod
         return DynamicAuthField.Image(label, bitmap, capability.sharedSecret)
     }
 
+    /**
+     * Extract text fields and radio groups from IdxRemediation.form.visibleFields.
```

Comment:
This is incomplete, this method does much more.

Reply:
updated, see if ok, else please suggest